user_manual/files/sharing.rst

```diff
@@ -27,16 +27,16 @@ A number of options are available for public *folder* shares:
 .. figure:: ../images/sharing_public_folder.png
 
 * **Read only** to allow viewing and downloading
-* ** Allow upload and editing**
-* With ** File drop**, the sharee can only upload files to a folder without seeing the files that are already in that folder.
```

Comment:
that is a person to which the folder is shared (https://en.wiktionary.org/wiki/sharee)
If that term is perhaps less know there might be a better word or some explanation when hovering over the term...

Reply:
I understand that word now. Well, we leave them unchanged.